bridges/otellogr/logsink_test.go

```diff
@@ -352,6 +352,40 @@ func TestLogSink(t *testing.T) {
 	}
 }
 
+// fix https://github.com/open-telemetry/opentelemetry-go-contrib/issues/6509
+func TestLogSinkCtxInInfo(t *testing.T) {
```

Comment:
Couldn't we add test cases to `TestLogSink` instead?

Reply:
Upon checking, the structure of `TestLogSink` is currently not suitable for my testing scenario. If adjustments are needed, more changes might be required.

Reply:
OK. Approving. I can look and try refactoring it tomorrow.